oauth2-code-flow-sample/NOTICE

```diff
@@ -0,0 +1,1823 @@
+Copyright (c) 2014-2015 Douglas Christopher Wilson <doug@somethingdoug.com>
+Copyright(c) 2009-2013 TJ Holowaychuk
+expressjs/mime-types 2.1.35 npmjs:mime-types/2.1.35
+Copyright (c) 2014 Jonathan Ong <me@jongleberry.com>
+Copyright (c) 2015 Douglas Christopher Wilson <doug@somethingdoug.com>
+Copyright(c) 2015 Douglas Christopher Wilson
+Copyright(c) 2014 Jonathan Ong
+finalhandler 1.2.0 npmjs:finalhandler/1.2.0
+Copyright(c) 2014-2022 Douglas Christopher Wilson
+Copyright (c) 2014-2022 Douglas Christopher Wilson <doug@somethingdoug.com>
+forwarded 0.2.0 npmjs:forwarded/0.2.0
+Copyright (c) 2014-2017 Douglas Christopher Wilson
+Copyright(c) 2014-2017 Douglas Christopher Wilson
+function-bind 1.1.1 npmjs:function-bind/1.1.1
+Copyright (c) 2013 Raynos.
+get-intrinsic 1.2.0 npmjs:get-intrinsic/1.2.0
+Copyright (c) 2020 Jordan Harband
+has 1.0.3 npmjs:has/1.0.3
+Copyright (c) 2013 Thiago de Arruda
+has-symbols 1.0.3 npmjs:has-symbols/1.0.3
+Copyright (c) 2016 Jordan Harband
+http-errors 2.0.0 npmjs:http-errors/2.0.0
+Copyright (c) 2016 Douglas Christopher Wilson doug@somethingdoug.com
+Copyright(c) 2016 Douglas Christopher Wilson
+Copyright(c) 2014 Jonathan Ong
+Copyright (c) 2014 Jonathan Ong me@jongleberry.com
+iconv-lite v0.4.24 npmjs:iconv-lite/0.4.24
+Copyright (c) 2011 Alexander Shtuchkin
+inherits v2.0.4 npmjs:inherits/2.0.4
+Copyright (c) Isaac Z. Schlueter
+jshttp/content-type 1.0.5 npmjs:content-type/1.0.5
+Copyright (c) 2015 Douglas Christopher Wilson
+Copyright(c) 2015 Douglas Christopher Wilson
+jshttp/fresh 0.5.2 npmjs:fresh/0.5.2
+Copyright(c) 2016-2017 Douglas Christopher Wilson
+Copyright (c) 2016-2017 Douglas Christopher Wilson <doug@somethingdoug.com>
+Copyright(c) 2012 TJ Holowaychuk
+Copyright (c) 2012 TJ Holowaychuk <tj@vision-media.ca>
+media-typer 0.3.0 npmjs:media-typer/0.3.0
+Copyright (c) 2014 Douglas Christopher Wilson
+Copyright(c) 2014 Douglas Christopher Wilson
+Copyright (c) 2014 Douglas Christopher Wilson
+Copyright(c) 2014 Douglas Christopher Wilson
+merge-descriptors 1.0.1 npmjs:merge-descriptors/1.0.1
+Copyright (c) 2015 Douglas Christopher Wilson <doug@somethingdoug.com>
+Copyright(c) 2015 Douglas Christopher Wilson
+Copyright(c) 2014 Jonathan Ong
+Copyright (c) 2013 Jonathan Ong <me@jongleberry.com>
+methods 1.1.2 npmjs:methods/1.1.2
+Copyright(c) 2015-2016 Douglas Christopher Wilson
+Copyright (c) 2015-2016 Douglas Christopher Wilson <doug@somethingdoug.com>
+Copyright (c) 2013-2014 TJ Holowaychuk <tj@vision-media.ca>
+Copyright(c) 2013-2014 TJ Holowaychuk
+mime 1.6.0 npmjs:mime/1.6.0
+Copyright (c) 2010 Benjamin Thomas, Robert Kieffer
+mime-db 1.52.0 npmjs:mime-db/1.52.0
+Copyright (c) 2014 Jonathan Ong <me@jongleberry.com>
+Copyright (c) 2015-2022 Douglas Christopher Wilson <doug@somethingdoug.com>
+Copyright(c) 2015-2022 Douglas Christopher Wilson
+Copyright(c) 2014 Jonathan Ong
+node-cookie-signature 1.0.6 npmjs:cookie-signature/1.0.6
+Copyright (c) 2012 LearnBoost "readmeFilename": "Readme.md"
```

Comment:
Everything in this line after LearnBoost can be removed.
This will make LearnBoost copyright lines duplicate, so you can deleted the following 2.